db/models.py

```diff
@@ -1,4 +1,12 @@
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
+from django.db.models import DateTimeField, ForeignKey, IntegerField, \
+    UniqueConstraint
+
+
+class User(AbstractUser):
+    ...
```

Comment:
The `User` model extends `AbstractUser`, which is a good approach for customizing the user model. Ensure that the `AUTH_USER_MODEL` setting in your Django settings is updated to use this custom `User` model if it is intended to replace the default one.